sdk/edgeorder/Azure.ResourceManager.EdgeOrder/src/autorest.md

```diff
@@ -0,0 +1,46 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+
+azure-arm: true
+csharp: true
+library-name: EdgeOrder
+namespace: Azure.ResourceManager.EdgeOrder
+require: https://github.com/Azure/azure-rest-api-specs/blob/58891380ba22c3565ca884dee3831445f638b545/specification/edgeorder/resource-manager/readme.md
+tag: package-2021-12
+output-folder: Generated/
+clear-output-folder: true
+mgmt-debug:
+  suppress-list-exception: true
```

Comment:
We should not use this flag, instead we should explicitly suppress them one by one